src/libsyncengine/requests/parameterscache.cpp

```diff
@@ -39,6 +39,13 @@ std::shared_ptr<ParametersCache> ParametersCache::instance(bool isTest /*= false
 }
 
 ParametersCache::ParametersCache(bool isTest /*= false*/) {
+    try {
+        ParmsDb::instance();
+    } catch (ParmsDb::ParmsDbIsNotInitializedException &) {
+        // ParmsDB is not initialized, behave as test
+        isTest = true;
+    }
```

Comment:
Tests are running in release mode on CI if I am right. So we could not use the build mode to choose behavior.
However, you are probably right that this might leads to wrongly initialized cache.
I will think about it thx

Reply:
Done in 5f03a8c293655a53017623455c1f6e5cbbcb917e
Test looks ok without it